feature/favourites/src/main/java/com/epmedu/animeal/favourites/FavouritesScreen.kt

```diff
@@ -0,0 +1,14 @@
+package com.epmedu.animeal.favourites
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.compose.runtime.getValue
+import androidx.hilt.navigation.compose.hiltViewModel
+
+@Composable
+fun FavouritesScreen() {
+    val viewModel = hiltViewModel<FavouritesViewModel>()
+    val state by viewModel.stateFlow.collectAsState()
+
+    FavouritesScreenUI(state) { viewModel.handleEvents(it) }
```

Comment:
It is better to do not move high-order functions out of parameters in compose. It would be nice to specify argument name to understand what this lambda does.

Reply:
moved to parameters